gapic/src/main/com/google/gapid/image/ArrayImage.java

```diff
@@ -723,7 +789,7 @@ public double getAlphaMax() {
 
     @Override
     public boolean isNormalized() {
-      return true;
+      return normalized;
```

Comment:
sigh... this really just duplicated `isHDR()` (well, the inverse, but still...). Feel free to leave as is, or incorporate with the change of `isHDR` and `isCount` into an enum as I'm requesting.

Reply:
Done